src/components/Commons/About.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+export const HeadSection = styled.div`
+  border-bottom: 1px solid #dfdfdf;
+  background: #f7f7f7;
+`;
+
+const Container = styled.div`
+  padding: 10%;
+  text-align: center;
+  font-family: 'Roboto', sans-serif;
+`;
+
+const Title = styled.h1`
+  font-size: 3rem;
+  font-weight: 100;
+  margin: 0 auto;
+  max-width: 62rem;
+`;
+
+const SubTitle = styled.p`
+  font-size: 1.4rem;
+  font-weight: 300;
+  line-height: 2.3rem;
+  margin: 1rem auto;
+  max-width: 32rem;
+  color: #414141;
+`;
+
+export const Header = ({ title, subtitle }) => (
```

Comment:
This component should be restructured. Take title, subtitle as props. and then render the children.
Eg : 
```
<div>
{props.title}
{props.subtitle}
{props.children}
</div>
```

Reply:
It would be more of a wrapper component. And it shouldn't be a shared component. It is a very specific use case for the About Section.

Reply:
@akshatnitd Where should this be placed? The About section components are not present in a single directory. We can either place the About section components together or place this wrapper component in Shared itself